### DISCUSSION TITLE
Add table_name folder in uploads project folder

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [nocodb/nocodb#406] [Features]
I ask you to consider the possibility when creating a column in the table Attachments create table_name folder in uploads project folder.
Project_name_folder/db/uploads/table_name_folder

**Comment by o1lab** [comment]
Have to see how this fits with various storage drivers. 

I see this is a good idea. And may be makes sense to add primary key in the name with random id appended to it like "primary_key_randomId.jpeg"
